src/applications/personalization/profile/components/personal-health-care-contacts/Instructions.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import HelpDeskContact from '../HelpDeskContact';
+const Instructions = ({ testId, contactType }) => {
+  const isEmergency = contactType === 'emergency contact';
 
-const Instructions = ({ testId }) => (
-  <div data-testid={testId}>
-    To add a contact, call us at <HelpDeskContact testId={testId} />.
-  </div>
-);
+  return (
+    <div data-testid={testId}>
+      To add {isEmergency ? 'an' : 'a'} {contactType}, call your VA health
+      facility.{' '}
+      <a testId={testId} to="https://www.va.gov/find-locations/">
+        Find your health facility’s phone number
+      </a>
```

Comment:
Same issue with this anchor tag needing an href.

Also for this, is the `testId` attribute being used in a unit test? It looks like that may have been confused with the `data-testid` attribute and also that would result in duplicate testId props being used in the same component, once on the outer div and once on the anchor tag. Maybe you can postfix the testId via template literal when it is used on the anchor? Something like this:
```js 
`${testId}-find-locations-link`
```
That would make it unique and individually queryable in your unit tests. Another option would be to get rid of that attribute on the anchor and use `getByText('Find your health facility’s phone number')` instead if you want to assert that the link renders appropriately.

Reply:
Thanks, I did try using <va-link /> but this caused the link to not display at all. I opted to switch back to the standard <a /> element.